source/_integrations/alexa.smart_home.markdown

```diff
@@ -221,33 +236,764 @@ alexa:
+      
+See the [troubleshooting](#troubleshooting) if for issues setting up the integration.
+
+
+#### Alexa Display Categories
+Configure a display category to override the display category and iconography each entity is shown in the Alexa app. This makes it easier to find and monitor devices. 
+
+```yaml
+light.kitchen_light:
+  display_categories: LIGHT,SWITCH
+```
+
+<div class='note info'>
+Devices such as cameras, doorbells, garage doors, and alarm control panels require specific display categories to provide all available features from Amazon Alexa. Overriding the display category provided by Home Assistant will limit features provided by Amazon Alexa. 
+</div> 
+
+See [Alexa Display Categories][alexa-display-categories] for a complete list
+  
+### Supported Integrations
+
+Home Assistant supports the following integrations through Alexa using a Smart Home Skill. For Home Assistant Cloud Users, documentation can be found [here](https://www.nabucasa.com/config/amazon_alexa/).
+
+The following integrations are currently supported:
+- [Alarm Control Panel](#alarm-control-panel)  
+- [Alert](#alert-automation-group-input-boolean)
+- [Automation](#alert-automation-group-input-boolean)
+- [Binary Sensor](#binary-sensor) 
+  - [Doorbell Announcement](#doorbell-announcement) 
+  - [Presence Detection](#presence-detection-with-binary-sensor)
+- [Climate](#climate)
+- [Cover](#cover)
+  - [Garage Doors](#garage-doors)
+- [Fan](#fan)
+  - [Fan Speed](#fan-speed)
+  - [Fan Direction](#fan-direction)
+  - [Fan Oscillation](#fan-oscillation)
+- [Group](#alert-automation-group-input-boolean)
+- [Input Boolean](#alert-automation-group-input-boolean)
+- [Input Number](#input-number)
+- [Image Processing](#image-processing)
+- [Light](#light)
+- [Lock](#lock)
+- [Media Player](#media-player)
+  - [Channels](#change-channel)
+  - [Speakers](#speaker-volume)
+  - [Sound Mode & Equalizers](#equalizer-mode)
+  - [Inputs](#inputs)
+  - [Payback Control](#seek)
+- [Scene](#scene)
+- [Script](#script)
+- [Sensor](#sensor)
+- [Switch](#switch)
+- [Timer](#timer)
+- [Vacuum](#vacuum)
+
+#### Alarm Control Panel
+Arm and disarm Alarm Control Panel entities. Ask Alexa for the state of the Alarm Control Panel entity.
+
+<kbd>
+
+_Alexa, arm my home in away mode._  
+_Alexa, arm my home._  
+_Alexa, disarm my home._  
+_Alexa, is my home armed?_  
+
+</kbd>
+
+##### Arming 
+Alarm Control Panel state must be in the `disarmed` state before arming. Alexa does not support switching from an armed state without first disarming. e.g. switching from `armed_home` to `armed_night`.
+
+The Alarm Control Panel state `armed_custom_bypass` isn't supported by Alexa and is treated as `armed_home`.
+
+<div class="note">
+
+Alexa does not support arming with voice PIN at this time. Therefore if the Alarm Control Panel requires a `code` for arming or the `code_arm_required` attribute is `true` the entity will not be exposed during discovery.  
+<br/>
```

Comment:
You cannot mix HTML & Markdown without having it surrounded by a new line. Or else the markdown parse will not switch language context.